src/Application/ZoomApplication.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+/*
+ * This file is part of the Secret Santa project.
+ *
+ * (c) JoliCode <coucou@jolicode.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JoliCode\SecretSanta\Application;
+
+use JoliCode\SecretSanta\Model\SecretSanta;
+use JoliCode\SecretSanta\Model\User;
+use JoliCode\SecretSanta\Zoom\MessageSender;
+use JoliCode\SecretSanta\Zoom\UserExtractor;
+use League\OAuth2\Client\Token\AccessToken;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+
+class ZoomApplication implements ApplicationInterface
+{
+    const APPLICATION_CODE = 'zoom';
```

Comment:
since you are using private const, I think it's better to be explicite about the visibiliy of theses 2 consts